mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+                            properties:
+                              - !ruby/object:Api::Type::String
+                                name: 'pattern'
+                                required: true
+                                description: |
+                                  Pattern defining the regular expression.
+                                  Its syntax (https://github.com/google/re2/wiki/Syntax) can be found under the google/re2 repository on GitHub.
+                              - !ruby/object:Api::Type::Array
+                                name: 'groupIndexes'
+                                description: |
+                                  The index of the submatch to extract as findings. When not specified, the entire match is returned. No more than 3 may be included.
+                                item_type: Api::Type::Integer
+                          - !ruby/object:Api::Type::NestedObject
+                            name: 'excludeInfoTypes'
+                            description: Set of infoTypes for which findings would affect
+                              this rule.
+                            properties:
+                              - !ruby/object:Api::Type::Array
+                                name: 'infoTypes'
+                                required: true
+                                description: |
+                                  If a finding is matched by any of the infoType detectors listed here, the finding will be excluded from the scan results.
+                                item_type: !ruby/object:Api::Type::NestedObject
+                                  properties:
+                                    - !ruby/object:Api::Type::String
+                                      name: 'name'
+                                      required: true
+                                      description: |
+                                        Name of the information type. Either a name of your choosing when creating a CustomInfoType, or one of the names listed
+                                        at https://cloud.google.com/dlp/docs/infotypes-reference when specifying a built-in type.
+                                    - !ruby/object:Api::Type::String
+                                      name: 'version'
+                                      description: |
+                                        Version of the information type to use. By default, the version is set to stable.
+                          - !ruby/object:Api::Type::NestedObject
+                            name: 'excludeByHotword'
+                            description: Drop if the hotword rule is contained in the proximate context.
+                            properties:
+                              - !ruby/object:Api::Type::NestedObject
+                                name: 'hotwordRegex'
+                                required: true
```

Comment:
Just want to double check if this field is required.

Reply:
`excludeByHotword` field works without `hotwordRegex` hence removing `required:true`